src/tools/manifest2proto.ts

```diff
@@ -343,12 +342,16 @@ export async function typeToProtoPayload(type: Type) {
     case 'Count': return {
       count: {}
     };
-    // TODO(b/154733929)
-    // case 'TypeVariable': return {
-    //   variable: {
-    //     name: (type as TypeVariable).variable.name,
-    //   }
-    // };
+    case 'TypeVariable': {
```

Comment:
serialization would be nicer, i.e. the following for typevars without constraints:
```
variable: {
  name: "a"
}
```

If you did something like this:
```
    case 'TypeVariable': {
      const constraint = type.canReadSubset || type.canWriteSuperset;
      return {
        variable: Object.assign({
          name: (type as TypeVariable).variable.name
        }, constraint ? {
          constraint: await typeToProtoPayload(constraint)
        } : {})
      };
    }
```

Reply:
Also, you may notice I've removed the `.resolvedType()` from the constraint option, as we encode the resolved typevars as their resolution.